CromIAM/src/main/scala/cromiam/cromwell/CromwellClient.scala

```diff
@@ -75,6 +102,8 @@ object CromwellClient {
 
   final case class CromwellConnectionFailure(f: Throwable) extends Exception(s"Unable to connect to Cromwell (${f.getMessage})", f)
 
+  final case class RootWorkflowMetadata(rootWorkflowId: Option[String])
```

Comment:
This doesn't seem used anywhere

Reply:
Yeah bats an artifact of an earlier attempt before I saw an easier path